reproject/utils.py

```diff
@@ -43,3 +43,159 @@ def parse_output_projection(output_projection, shape_out=None):
             raise ValueError("Need to specify shape when specifying output_projection as WCS object")
 
     return wcs_out, shape_out
+
+def flatten(f):
+    """Flatten a radio FITS continuum 'cube' so that it becomes a 2D
+    image suitable for reprojection. Return new PrimaryHDU"""
+
+    naxis=f[0].header['NAXIS']
+    if naxis<2:
+        raise Exception('Can\'t make map from this')
+    if naxis==2:
+        return f[0].header,f[0].data
+
+    w = WCS(f[0].header)
+    wn=WCS(naxis=2)
+    
+    wn.wcs.crpix[0]=w.wcs.crpix[0]
+    wn.wcs.crpix[1]=w.wcs.crpix[1]
+    wn.wcs.cdelt=w.wcs.cdelt[0:2]
+    wn.wcs.crval=w.wcs.crval[0:2]
+    wn.wcs.ctype[0]=w.wcs.ctype[0]
+    wn.wcs.ctype[1]=w.wcs.ctype[1]
+    
```

Comment:
would you mind adding white spaces between operators like ``=`` and after ``,``?